core/src/main/resources/jenkins/formelementpath/form-element-path.js

```diff
@@ -40,24 +44,23 @@ document.addEventListener("DOMContentLoaded", function(){
             return m;
         }
 
-        var jsonElement = null;
-
-        for (var i = 0; i < form.elements.length; i++) {
+        for (let i = 0; i < form.elements.length; i++) {
             var e = form.elements[i];
             if (e.name == "json") {
-                jsonElement = e;
                 continue;
             }
             if (e.tagName == "FIELDSET")
-                continue;
+                {continue;}
```

Comment:
Unfortunate?

Reply:
I manually fixed all the ones I found, it's what eslint did with the `--fix`. I will fixed this